### PR TITLE
debug_pipeline

### PR DESCRIPTION
**Pull request by jkobject**
after testing the loading process and the post processing of all datatypes

### CODE REVIEW

File: depmapomics/config.py [hunk 1]
```diff
@@ -89,7 +121,7 @@
 rnasource7 = "ibm"
 
 ## our working workspace (reference)
-rnaworkspace = "broad-firecloud-ccle/DepMap_hg38_RNAseq"
+RNAWORKSPACE = "broad-firecloud-ccle/DepMap_hg38_RNAseq"
 
 ## curent WGS GP buckets
 wgsworkspace1 = "terra-broad-cancer-prod/DepMap_WGS"
```
Comment (javadnoorb) [inline depmapomics/config.py:127]: maybe capitalize this as well?

**Review by javadnoorb — COMMENTED** [review]
@jkobject  is this PR mainly refactoring?

**Comment by jkobject** [comment]
its mainly debug

File: depmapomics/config.py (hunk 1 above)
Comment (jkobject) [reply depmapomics/config.py:127]: I could, I wanted to leave them uncapitalize to really see the distinction with the compute workspace / gp data workspaces.

**Comment by javadnoorb** [comment]
Looks like there are some merge conflicts. Otherwise look good to me.

**Pull request merged**